test/javascripts/components/load-more-test.js.es6

```diff
@@ -0,0 +1,28 @@
+import { _setTestParams } from "discourse/lib/eyeline";
+import componentTest from "helpers/component-test";
+
+moduleForComponent("load-more", { integration: true });
+
+componentTest("preview updates with markdown", {
+  template: `
+    {{#load-more selector=".numbers tr" action=(action loadMore)}}
+      <table class='numbers'>
+        <tr><td>1</td></tr>
```

Comment:
do we need all the lines to test the behavior?

Reply:
I significantly simplified the table to keep a single empty `tr`.